astraguard/swarm/response_orchestrator.py

```
@@ -409,6 +409,30 @@ async def _execute_constellation(
+                        scope=ActionScope.CONSTELLATION.value,
+                    )
+                    
+                    if not is_safe:
+                        logger.warning(
+                            f"CONSTELLATION action blocked by safety simulator: "
+                            f"{decision.action}"
+                        )
+                        self.metrics.safety_gate_blocks += 1
+                        return False
+                        
+                except Exception as sim_error:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)